### PR TITLE
Allow PHP8

### PR DESCRIPTION
**Pull request by T2L**

### CODE REVIEW

**Comment by T2L** [comment]
Alright, the last thing that is missing https://github.com/phpmd/phpmd, which is not PHP8 compatible because one of it's dependencies (https://github.com/pdepend/pdepend) is not compatible, but work is in progress: https://github.com/pdepend/pdepend/pull/498

**Review by michalbundyra — COMMENTED** [review]
It looks good! Thanks for your effort.

I think we can ignore phpmd issues on PHP 8.0 for now.

Would you be able to rebase and update also phpunit workflow to use PHP 8.0? I've added github actions recently, and at some point we will drop Travis.

I've tried to push to your branch, but it looks like you've locked that. Thanks again

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ cache:
     - $HOME/.composer/cache
 
 php:
+  - nightly
```
[the remark this answers is not in the document]
Comment (T2L) [reply .travis.yml]: PHP8.0 was not available as `8.0` at the time of this PR, but now it is https://travis-ci.community/t/php-8-0-missing/10132/27

**Comment by T2L** [comment]
- The fork has been synced and rebased. I think you cannot push to it because the repo is part of the organization and not my personal Github
- Changed `nightly` to `8.0` on Travis
- Added `8.0` to Github Actions (BTW, even `8.1` can be added)

**Comment by michalbundyra** [comment]
Thanks, @T2L!

**Comment by T2L** [comment]
Any news regarding the release plan?

**Comment by michalbundyra** [comment]
Yes, later today will be released :)

**Pull request merged**